### PR TITLE
Urls: don’t allow javascript protocol followed by url stub to match

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by ingluisjimenez** [comment]
👍

File: src/Validation/Rules/UrlRule.php [hunk 1]
```diff
@@ -11,4 +11,19 @@ class UrlRule extends RegexRuleAbstract {
 
   protected $_pattern = '/^((?:https?):\/\/)?(?:(?:(?:[\w\.\-\+!$&\'\(\)*\+,;=_]|%[0-9a-f]{2})+:)*(?:[\w\.\-\+%!$&\'\(\)*\+,;=_]|%[0-9a-f]{2})+@)?(?:[A-Za-z0-9_\-]+\.)(?:[A-Za-z0-9_\-\.])+(?::[0-9]+)?(?:[\/|\?](?:[\w#!:\.\?\+=&@$\'~*,;_\/\(\)\[\]\-]|%[0-9a-f]{2})*)?$/ui';
 
+  /**
+   * @inheritDoc
+   */
+  public function isValid( $data, array $context = null ) {
+
+    $is_valid = parent::isValid( $data, $context );
```
Comment (jimdelois) [inline src/Validation/Rules/UrlRule.php]: You can just kill this temp.

**Comment by TJP85** [comment]
👍

**Comment by bryanlatten** [comment]
👍

File: src/Validation/Rules/UrlRule.php (hunk 1 above)
Comment (mikesherov) [inline src/Validation/Rules/UrlRule.php]: Good catch.

**Comment by jimdelois** [comment]
👍

**Pull request merged**